decidim-admin/app/views/decidim/admin/officializations/index.html.erb

```diff
@@ -18,6 +18,12 @@
         </li>
       </ul>
   </div>
+
+  <div class="column medium-3">
```

Comment:
the css `row` class doesn't seem to work. I can change it to `card-grid` and then it works. Can you check?
(I've also separated the search input into another row.

Reply:
ok, I've checked it. Now using:

```
<div class="filters grid-x">
  <div class="cell medium-2">...()... </div>
  <div class="cell medium-2">...</div>
  <div class="cell medium-8">...</div>
</div>
```
But this can be improved/refactored in future iterations when doing the whole filter-box.

![Captura de pantalla de 2019-11-15 15-25-53](https://user-images.githubusercontent.com/210216/68950337-3ef96800-07bc-11ea-83cb-af93cd274fcd.png)